.gitignore

```diff
@@ -62,3 +62,4 @@ ehthumbs.db
 Thumbs.db
 
 
+/Assets/Mirror/Plugins/Unity.Cecil.meta
```

Comment:
you can remove this.  
Instead,  remove your empty folder /Assets/Mirror/Plugins/Unity.Cecil

Reply:
Done